Gemfile

```diff
@@ -57,7 +57,7 @@ gem 'webrick', '>= 1.7.0' if RUBY_VERSION >= '3.0.0' # No longer bundled by defa
 gem 'yard', '~> 0.9'
 
 if RUBY_VERSION >= '2.4.0'
-  gem 'rubocop', '~> 1.10', require: false
+  gem 'rubocop', ['~> 1.10', '< 1.33.0'], require: false
```

Comment:
```suggestion
  gem 'rubocop', '~> 1.32.0', require: false
```

Reply:
> I tried to do that with ~> 1.32.0 , but bundler could not find compatible versions for old rubies…. and I don’t wanna spend time finding the correct version just to unblock our release…

Missed that part. Maybe add a comment describing that, or scope Rubocop to a more recent Ruby version.

Reply:
FYI Rubocop `1.32` needs Ruby 2.6: https://rubygems.org/gems/rubocop/versions/1.32.0